src/app/index.scss

```diff
@@ -67,16 +67,39 @@ h2 {
 }
 
 .data-card {
+  color: white;
   background-color: #38425D;
-  h2 {
-    color: white;
+
+  .sidebar-header {
+    padding-bottom: 5px;
+    border-bottom: 1px solid #7883A0;
+    margin-bottom: 5px;
   }
-  a {
-    color: white;
+
+  .current-dataset {
+    display: inline-block;
+    white-space: nowrap;
+    width: 70%;
+    overflow: hidden;
+    text-overflow: ellipsis;
   }
   a:hover, a:focus {
     color: #ddd;
   }
+  .small-button {
```

Comment:
I chose not to because it looked incongruous with the rest of the new Ui, which is more flat—I followed an inverted variant of the big buttons Jim implemented for the welcome modal.